### PR TITLE
NPE on search

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

**Review by makosblade — APPROVED** [review]

File: src/main/java/com/openlattice/datastore/services/DatastoreElasticsearchImpl.java [hunk 1]
```diff
@@ -1508,12 +1508,13 @@ public void verifyRunner() throws UnknownHostException {
             UUID entitySetId,
             Map<UUID, DelegatedUUIDSet> authorizedPropertyTypesByEntitySet ) {
         Map<UUID, Map<String, Float>> fieldsMap = Maps.newHashMap();
-        authorizedPropertyTypesByEntitySet.get( entitySetId ).forEach( propertyTypeId -> {
-            String fieldName = getFieldName( propertyTypeId );
+        authorizedPropertyTypesByEntitySet.getOrDefault( entitySetId, DelegatedUUIDSet.wrap( ImmutableSet.of() ) )
```
Comment (geekbeast) [inline src/main/java/com/openlattice/datastore/services/DatastoreElasticsearchImpl.java:1511]: Do you need to wrap?

**Review by geekbeast — COMMENTED** [review]
Quick question.

File: src/main/java/com/openlattice/datastore/services/DatastoreElasticsearchImpl.java (hunk 1 above)
Comment (katherinebernstein) [reply src/main/java/com/openlattice/datastore/services/DatastoreElasticsearchImpl.java:1511]: I didn't. see a. way to. do like DelegatedUUIDSet.empty() or something like that :/

Comment (geekbeast) [reply src/main/java/com/openlattice/datastore/services/DatastoreElasticsearchImpl.java:1511]: I'm pretty sure that putting Set<UUID> would be fine, unless this gets serialized into an IMap at some point.

**Review by geekbeast — APPROVED** [review]
It should be fine for now, pretty minor.

**Pull request merged**